scraper.rb

```diff
@@ -9,33 +9,25 @@
 require 'open-uri/cached'
 OpenURI::Cache.cache_path = '.cache'
 
-BASE = 'http://www.parliament.gov.zm'
-
 def scrape(h)
   url, klass = h.to_a.first
   klass.new(response: Scraped::Request.new(url: url).response)
 end
 
 ScraperWiki.sqliteexecute('DELETE FROM data') rescue nil
 
-# We should really extract these from the 'Next' links
-pages = [
-  '/members-of-parliament',
-  '/members-of-parliament/page/1/0',
-  '/members-of-parliament/page/2/0',
-]
-
-added = 0
-pages.each do |page|
-  url = URI.join(BASE, page).to_s
-  warn "Fetching #{url}"
-  (scrape url => MembersPage).member_rows.each do |row|
+def scrape_members(url)
```

Comment:
Great, thanks. The other slight issue here is that you've given this a command name, when it's really a query function. We're not 100% consistent in this everywhere, but it would be good to become so, so we can more easily see at a glance when a function is simply gathering data for something else to use (e.g. to save to SQLite separately), vs when it's taking care of all that itself.